packages/connect-popup/src/static/log.html

```diff
@@ -0,0 +1,50 @@
+<!-- TODO: here we could have some vanilla javascript/html that will display the logs that are collected by logs shared worker -->
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+        <meta
+            name="viewport"
+            content="width=device-width,initial-scale=1.0,maximum-scale=1.0,user-scalable=0"
+        />
+        <title>TrezorConnect | Trezor</title>
+        <meta name="description" content="" />
+        <meta name="keywords" content="" />
+        <meta name="author" content="Trezor info@trezor.io" />
+        <meta name="robots" content="noindex, nofollow" />
+        <meta name="title" content="Trezor Connect" />
+        <meta name="theme-color" content="#ffffff" />
+        <meta http-equiv="Pragma" content="no-cache" />
+        <meta http-equiv="Expires" content="-1" />
+        <link href="./popup.css" rel="stylesheet" />
+
+        <!-- suite-like fonts -->
+        <link media="all" rel="stylesheet" href="./fonts/fonts.css" />
+    </head>
+    <body>
+        <main>
+            <!--
+                give react slot min-width of left panel (InfoPanel) which is not
+                loaded yet to prevent loader from "jumping"
+             -->
+            <div id="react" style="min-width: 35vw"></div>
+            <section id="container">
+                <h2>Logger from connect</h2>
+            </section>
+        </main>
+
+        <script type="text/javascript" async="false">
+            // import LogWorker from './sharedLoggerWorker';
```

Comment:
imho if you have static file name, you should be able to reference it in vanilla js
new SharedWorker('./build/somethig.js') and then only bind onmessage handlers and print them somewhere. vanilla js should be enough here.

Reply:
Done and works!